### PR TITLE
Apache Kafka attribution

### PR DESCRIPTION
**Pull request by k-wall**
### Type of change

- Documentation

### Description

Improve Apache Kafka attribution in the repo's readme.

### Additional Context

_Why are you making this pull request?_

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Write tests
- [ ] Make sure all tests pass
- [ ] Review performance test results. Ensure that any degradations to performance numbers are understood and justified.
- [ ] Make sure all Sonarcloud warnings are addressed or are justifiably ignored.
- [ ] Update documentation
- [ ] Reference relevant issue(s) and close them after merging
- [ ] For user facing changes, update CHANGELOG.md (remember to include changes affecting the API of the test artefacts too).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Kroxylicious
 
-Kroxylicious, the snappy open source proxy for Apache Kafka
+Kroxylicious, the snappy open source proxy for [Apache Kafka®](https://kafka.apache.org).
```
Comment (robobario) [inline README.md:3]: Do we only need the ® on the first mention in the page? we have another mention of Kafka at line 5.

**Review by robobario — APPROVED** [review]
LGTM

**Comment by SamBarker** [comment]
I pushed up https://github.com/kroxylicious/kroxylicious/pull/1410 before seeing this was open which was based on a search and replace approach.

File: README.md (hunk 1 above)
Comment (SamBarker) [reply README.md:3]: First/main usage is the official ask yes.

**Review by SamBarker — APPROVED** [review]

**Pull request merged**